Evaluation/workflow.R

```diff
@@ -59,7 +60,7 @@ createQualificationReport <- function(qualificationRunnerFolder,
   #' `workingDirectory`: current directory is used as default working directory
   workingDirectory <- getwd()
 
-  qualificationPlanName <- "Evaluation_plan.json"
+  qualificationPlanName <- "evaluation_plan.json"
   qualificationPlanFile <- file.path(workingDirectory, "Input", qualificationPlanName)
```

Comment:
Just FYI, in the template from here https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/articles/qualification-workflow.html#template
the folder is `input` and not `Input`, and the GHA checks fail.

Reply:
Where are the failing GHA checks?

Reply:
Maybe we should change the template then :)
But the folder is also configurable in the GHA